pkg/apis/pipeline/v1beta1/taskrun_types_test.go

```diff
@@ -217,50 +216,6 @@ func TestTaskRunHasStarted(t *testing.T) {
 	}
 }
 
-func TestTaskRunIsOfPipelinerun(t *testing.T) {
```

Comment:
why not move these into metrics_test.go?

Reply:
currently, the same scenarios are tested through the public API of the `taskrunmetrics` package.

I also felt that adding a test for a un-exported function would make the tests a little brittle, as I feel the implementation should be able to change below the public API without modifying the tests.

Im happy to add specific tests for this function, but would prefer to also then make it exported if we go that path.

Reply:
ok that makes sense-- it does seem like there's test coverage [here](https://github.com/tektoncd/pipeline/blob/c245f3ecf9eafae7ccef1afdefbe234093baf438/pkg/taskrunmetrics/metrics_test.go#L275)